client/tracker.go

```diff
@@ -0,0 +1,209 @@
+package client
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math/rand"
+	"net"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/matei-oltean/go-torrent/fileutils"
+)
+
+// the actions for an udp transfer
+const (
+	aConnect uint32 = iota
+	aAnnounce
+	aScrap
+	aError
+)
+
+// expectedPeers is the number of peers we are requesting
+const expectedPeers int = 20
+
+// getPeers returns the list of peers from a torrent file and client ID
+func getPeers(torrentFile *fileutils.TorrentFile, clientID [20]byte) ([]string, error) {
```

Comment:
Move it to fileUtils and make change it to `func (t *TorrentFile) getPeers ...` instead?